### PR TITLE
Implement abstracted support for cancelling cluster tasks

### PR DESCRIPTION
**Pull request by nickbp**
Bump scheduler coverage by a few percent

### CODE REVIEW

File: cli/dcos-cassandra/main.go [hunk 1]
```diff
@@ -151,6 +151,10 @@ func (cmd *BackupRestoreHandler) runBackup(c *kingpin.ParseContext) error {
 	cli.PrintJSON(cli.HTTPPutJSON("v1/backup/start", string(payload)))
 	return nil
 }
+func (cmd *BackupRestoreHandler) runBackupStop(c *kingpin.ParseContext) error {
```
Comment (triclambert) [inline cli/dcos-cassandra/main.go]: What about stopping the other operations?

Comment (nickbp) [inline cli/dcos-cassandra/main.go]: Not opposed to the idea, didn't know enough about the context of these operations to know whether it made sense. I think it'd be straightforward to add. Any preference Mohit?

Comment (nickbp) [inline cli/dcos-cassandra/main.go]: Done.

**Comment by nickbp** [comment]
retest this please

**Comment by kow3ns** [comment]
LGTM ... Like the abstraction and generalization implemented in ClusterTaskRunner
Can we update the documentation for the stop commands in this branch prior to merging?

**Comment by nickbp** [comment]
Done

**Comment by kow3ns** [comment]
retest this please

**Pull request merged**